huaweicloud/services/cdn/resource_huaweicloud_cdn_domain_rule.go

```diff
@@ -49,28 +50,27 @@ func ResourceCdnDomainRule() *schema.Resource {
 			"name": {
 				Type:        schema.TypeString,
 				Required:    true,
-				Description: "Specifies the domain name.",
+				Description: "The domain name.",
 			},
 			"rules": {
-				Type:        schema.TypeSet,
-				Required:    true,
-				Description: "Specifies a list of rules.",
+				Type:     schema.TypeSet,
+				Required: true,
 				Elem: &schema.Resource{
 					Schema: map[string]*schema.Schema{
 						"name": {
 							Type:        schema.TypeString,
 							Required:    true,
-							Description: "Specifies the rule name. The valid length is limit from `1` to `50`.",
+							Description: "The rule name. The valid length is limit from `1` to `50`.",
```

Comment:
```suggestion
							Description: "The rule name.",
```
Just only specifies the introduction

Reply:
done